pymathics/natlang/normalization.py

```diff
@@ -0,0 +1,301 @@
+"""
+
+Text normalization
+
+"""
+import itertools
+from itertools import islice
+from typing import Optional
+
+import spacy
+from mathics.core.atoms import Integer, String
+from mathics.core.convert.python import from_python
+from mathics.core.evaluation import Evaluation
+from mathics.core.list import ListExpression
+
+from pymathics.natlang.spacy import _cases, _pos_tags, _position, _SpacyBuiltin
+
```

Comment:
We need a Python statement like:

```
sort_order = "Text normalization" 
```

Currently in the summary list we have: 


* Linguistic Data
* Text normalization
* Text analysis functions
* Language translation

"Text Normalization" appears before "Language translation" because of the order of the modules: "normalization" appears before "translation".

Similarly for the other modules.